tox.embedapi.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = sphinx-{18,20,21,22,23,24,30,31,32,33,34,35,40,41,latest}
+envlist = sphinx-{18,20,21,22,23,24,30,31,32,33,34,35,40,41,42,latest}
```

Comment:
Hopefully someday we can delete some of these :D

Reply:
Yeah, I'd like to remove at least 1.x and 2.x; but there are a lot of people that still depend on them --so, I prefer to keep running tests on these versions at least until we start finding issues and making them harder to support.